src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -503,24 +503,836 @@ A table with a full reference of error codes, messages and description:
+|`scheduledBefore`|`DateTime`| | Filter for events which were scheduled up until this date.|
+|`updatedAfter`|`DateTime`| | Filter for events which were updated after this date. Cannot be used together with `updatedWithin`.|
+|`updatedBefore`|`DateTime`| | Filter for events which were updated up until this date. Cannot be used together with `updatedWithin`.|
+|`updatedWithin`|`Duration`| | Include only items which are updated within the given duration.<br><br> The format is [ISO-8601#Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations)|
+|`skipMeta`|`Boolean`| `true`&#124;`false` | Exclude the meta data part of response (improves performance)|
+| TODO (?) `dataElementIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`|	Data element ID scheme to use for export.|
+| TODO (?) `categoryOptionComboIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`| Category Option Combo ID scheme to use for export|
+| TODO (?) `orgUnitIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`| Organisation Unit ID scheme to use for export|
+| TODO (?) `programIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`| Program ID scheme to use for export|
+| TODO (?) `programStageIdScheme`|`String`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`| Program Stage ID scheme to use for export|
+|`idScheme`|`string`| `UID`&#124;`CODE`&#124;`ATTRIBUTE:{ID}`| Allows to set id scheme for data element, category option combo, orgUnit, program and program stage at once.|
+|`order`|`String`|comma-delimited list of `OrderCriteria` in the form of `propName:sortDirection`.<br><br> Example: `createdAt:desc`<br><br>**Note:** `propName` is case sensitive, `sortDirection` is case insensitive|Sort the response based on given `OrderCriteria`|
+|`event`|`String`|comma-delimited list of `uid`| Filter the result down to a limited set of IDs by using event=id1;id2.|
+|`skipEventId`|`Boolean`| |	Skips event identifiers in the response|
+|`attributeCc` (see note)|`String`| Attribute category combo identifier (must be combined with attributeCos)|
+|`attributeCos` (see note)|`String`| Attribute category option identifiers, separated with ; (must be combined with attributeCc)|
+|`includeDeleted`|`Boolean`| |	When true, soft deleted events will be included in your query result.|
+|`assignedUserMode`|`String`| `CURRENT`&#124;`PROVIDED`&#124;`NONE`&#124;`ANY`| Assigned user selection mode|
+|`assignedUser`|`String`|comma-delimited list od `uid`| Filter the result down to a limited set of events that are assigned to the given user IDs by using `assignedUser=id1;id2`.<br><br>This parameter will be considered only if assignedUserMode is either `PROVIDED` or `null`.<br><br>The API will error out, if for example, `assignedUserMode=CURRENT` and `assignedUser=someId`|
+
+> **Note**
+>
+> If the query contains neither `attributeCC` nor `attributeCos`, the server returns events for all attribute option combos where the user has read access.
```

Comment:
If no AoC or CoC are defined for the program/program stage, the default AoC and CoC are used, so this info is just relevant for people actively using AoC and/or CoC. 

I think the explanation you have is good, just adding some extra info for you :)